android/build.gradle

```diff
@@ -2,13 +2,11 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "33.0.0"
+        buildToolsVersion = "34.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 33
-        targetSdkVersion = 33
-
-        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
-        ndkVersion = "23.1.7779620"
+        compileSdkVersion = 34
+        targetSdkVersion = 34
```

Comment:
Bumping `targetSdkVersion` to 34 caused crash on android 14 devices.
The root cause was in react-native-blob-utils package, which we had to bump version to fix it.